src/isa/riscv64/system/priv.c

```diff
@@ -801,24 +804,24 @@ static word_t priv_instr(uint32_t op, const rtlreg_t *src) {
           if(cpu.v) longjmp_exception(EX_VI);
           if(cpu.mode == MODE_U) longjmp_exception(EX_II);
           if(!(cpu.mode == MODE_M || (cpu.mode == MODE_S && !cpu.v))) longjmp_exception(EX_II);
-          mmu_tlb_flush(*src);
+          mmu_vmtlb_flush(*src);
           break;
         case 0x31: // hfence.gvma
           if(cpu.v) longjmp_exception(EX_VI);
           if(cpu.mode == MODE_U) longjmp_exception(EX_II);
           if(!(cpu.mode == MODE_M || (cpu.mode == MODE_S && !cpu.v && mstatus->tvm == 0))) longjmp_exception(EX_II);
-          mmu_tlb_flush(*src);
+          mmu_vmtlb_flush(*src);
           break;
 #ifdef CONFIG_RV_SVINVAL
         case 0x13: // hinval.vvma
           if(cpu.v) longjmp_exception(EX_VI);
           if(cpu.mode == MODE_U) longjmp_exception(EX_II);
-          mmu_tlb_flush(*src);
+          mmu_vmtlb_flush(*src);
```

Comment:
Read the function of  "hinval.vvma" in _the RISC-V Instruction Set Manual
Volume II: Privileged Architecture_